keys.go

```diff
@@ -83,6 +95,17 @@ type Keys struct {
 	Keys     []Key        `json:"resources"`
 }
 
+type KeyVersionsMetadata struct {
+	CollectionType  string `json:"collectionType"`
+	CollectionTotal int    `json:"collectionTotal"`
```

Comment:
Optionally we can make CollectionTotal as pointer to be consistent with other int values in request/response.